process/security_baseline.md

```diff
@@ -0,0 +1,253 @@
+## Overview
+OpenSSF’s mission is to make open source software more secure. The OpenSSF Security Baseline is one of the key initiatives to achieve this mission. The baseline establishes the minimum practical security standards for OpenSSF software projects throughout various lifecycle stages, ensuring a strong Minimum Viable Secure Product (MVSP) across Linux Foundation projects and internal work.
+
+The existing OpenSSF guides and technologies will be used to frame the baseline, define objectives, provide implementation recommendations, and suggest verification methods. Through community engagement and collaboration, the baseline will first be adopted by a few software-based pilot projects before being fully adopted by all OpenSSF projects.
+
+The baseline provides a foundational framework for systematic adoption across the Linux Foundation. Collaboration with peer foundations is essential for baseline customization and adoption, aiming to enhance the security of the open source software ecosystem.
+
+As technology and the threat landscape evolve, the baseline will continuously adapt. The primary goal of this initiative is to maintain a balance between security, reliability, performance, and cost-effectiveness.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119/).
+
+## Background
+The initiative was one of the outcomes of the October 2023 Linux Foundation Member Summit. Making open source software more secure is one of the top priorities across the Linux Foundation. OpenSSF is leading the charge. 
+
+In the United States, open source software is used across all critical infrastructure sectors defined by CISA (Cybersecurity and Infrastructure Security Agency), for example, health care, defense, financial services, utilities, telecommunications, etc. Open source security directly impacts national security, economics and social stability. Enhancing open source security is imperative. NIST has published a [Secure Software Development Framework](https://csrc.nist.gov/Projects/ssdf) (SSDF) as a result of Executive Order (EO) 14028 on "[Improving the Nation's Cybersecurity](https://www.federalregister.gov/documents/2021/05/17/2021-10460/improving-the-nations-cybersecurity)".
+
+In the European Union, [Cyber Resiliency Act](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CONSIL:ST_17000_2023_INIT) (CRA) has declared a new role - Open Source Steward(Article 3(18a). The legislation is to ensure that consumers of products with digital features are protected, and manufacturers of these products are held accountable for product security. Open source security is the foundation for manufacturers to be compliant with the legislation.     
```

Comment:
I don't know if this is how the CRA text is written, but the term "manufacturer" isn't typically used in the context of software. In SLSA, for example, the term we have agreed upon is the "software producer", so I would recommend using a term like "software producer" whenever referring to the entity that creates a software product.

Reply:
> I don't know if this is how the CRA text is written, but the term "manufacturer" isn't typically used in the context of software. In SLSA, for example, the term we have agreed upon is the "software producer", so I would recommend using a term like "software producer" whenever referring to the entity that creates a software product.

Thanks @marcelamelara I will use software producer in other places. In CRA context, manufacturer needs to be used. I had to update my presentation to replace producer with manufacturer when talking about OpenSSF in light of CRA

Reply:
Interesting, thanks for clarifying.